static/js/dashboard.js

```diff
@@ -186,7 +186,10 @@ async function updateDashboard() {
                 showToast('Dashboard Updated Successfully');
                 return true;
             }
-            return res.json();
+            return res.json().then(err => {
+                showToast(err.message);
```

Comment:
could you prepend something like:
`request failed: ..whatever error backend returned....`

Reply:
done 

<img width="1464" alt="image" src="https://github.com/siglens/siglens/assets/113487319/8f645ae3-802a-453b-a909-c554ad15809e">